scipy/signal/fir_filter_design.py

```diff
@@ -594,3 +596,106 @@ def remez(numtaps, bands, desired, weight=None, Hz=1, type='bandpass',
     bands = np.asarray(bands).copy()
     return sigtools._remez(numtaps, bands, desired, weight, tnum, Hz,
                            maxiter, grid_density)
+
+
+def firls(numtaps, bands, desired, weight=None, fs=2):
```

Comment:
I was thinking that the IIR filter design functions and freqz etc could be harmonized by adding an fs= parameter, which would make it easier to enter frequencies without thinking about whether they're in cycles/sample, half-cycles/sample, radians/sample, etc.